rust/private/rustc.bzl

```diff
@@ -191,8 +235,6 @@ def rustc_compile_action(
     if ar_str.find("libtool", 0) != -1:
         ar = "/usr/bin/ar"
```

Comment:
Is this old `ar` finding logic obsolete with your changes?

Reply:
Looks like this can be removed

```
>> rustc --version
rustc 1.26.1 (827013a31 2018-05-25)

>> rustc -C help
...
    -C                ar=val -- this option is deprecated and does nothing
```